daemon/main.c

```diff
@@ -400,11 +400,26 @@ static const char *verify_required_directory(const char *dir) {
 #ifdef ENABLE_HTTPS
 static void security_init(){
     char filename[FILENAME_MAX + 1];
-    snprintfz(filename, FILENAME_MAX, "%s/ssl/key.pem",netdata_configured_user_config_dir);
-    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  filename);
+    char common[8];
```

Comment:
We do we need this array? Can't we use the string literal `"none"`?

Reply:
I used this to be sure that compiles would not allocate memory, instead they would use the `stack`. I confess I am few years without to do reverse engineer, so I am not sure that this "problem" was already solved, but I will remove it as you suggested. I agree that for a small text this is not so necessary.

Reply:
The last commit removed the array.